editor/src/third-party/react-error-overlay/components/CodeBlock.tsx

```diff
@@ -30,7 +30,7 @@ const secondaryPreStyle = {
   backgroundColor: yellowTransparent,
 }
 
-const codeStyle = {
+export const codeStyle = {
```

Comment:
I think this should not be exported as this is source code of react-error-overlay, and instead, if there's no relevant entry, add one to `UtopiaStyles`

Reply:
👍